Packs/CommonScripts/Scripts/IsUrlPartOfDomain/IsUrlPartOfDomain.py

```diff
@@ -0,0 +1,37 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+def main(domain_name: str, urls: str) -> CommandResults:
+    """Checks that the urls are in the domain in the domain name.
+
+    Args:
+        domain_name: A domain
+        urls: URLs to check if the domain in them.
+
+    Returns:
+        Results to display in CortexXSOAR
+    """
+    urls = argToList(urls)
+    results = demisto.executeCommand('ExtractDomainFromUrlAndEmail', {'input': domain_name})
+    domain = results[0]['Contents']
+
+    if not domain:
+        return_error(f'Could not find a domain in "{domain_name}"')
+    domain = domain.lower()
+
+    outputs: list = list()
+    for url in urls:
+        results = demisto.executeCommand('ExtractDomainFromUrlAndEmail', {'input': url.lower()})
```

Comment:
dont run the script for each, just pass the csv

Reply:
But then I'll get multiple domains so it's open to bugs.